docs/api-testing/use-cases/fact.md

```diff
@@ -0,0 +1,279 @@
+- In the Override section, add one schedule adding `env` as key and `production` as value: in this way, the test will be executed with `production` value as environment.
+  :::
+
+## Disabling Email Notifications
+
+The **Fact** component is also usable for disabling email notifications:
+
+```yaml
+  - id: fact
+    identifier: disable_alerts
+    label: alerts disabled
+    value: "true"
+```
+
+<img src={useBaseUrl('img/api-testing/factDisableAlert.png')} alt="factDisableAlert.png" />
+
+You can use logic in the test to set the Fact component and use that to alter the email notification.
+
+As an example, you could say "IF the env is development, then disable emails for this test":
+
+<img src={useBaseUrl('img/api-testing/factAlertDisabled.png')} alt="factAlertDisabled.png" />
+
+:::warning Important
+`Identifier` must be equal to `disable_alerts`.
+:::
+
+### Use Case: Disabling Email Notifications
+
+1. In the composer, add the **Fact** component.
+
+<img src={useBaseUrl('img/api-testing/factDisableAlert.png')} alt="factDisableAlert.png" />
+
+```yaml
+  - id: fact
+    identifier: disable_alerts
+    label: alerts disabled
+    value: "true"
+```
+
+2. Add the **GET** request.
+
+   - Url - for example `http://demoapi.apifortress.com/api/retail/product`
+   - Variable - for example `payload`
+   - Mode - for example `json`
+
+<img src={useBaseUrl('img/api-testing/fact-examples/get.png')} alt="get request" />
+
+```yaml
+  - id: get
+    children: []
+    url: http://demoapi.apifortress.com/api/retail/product
+    var: payload
+    mode: json
+```
+
+3. **Save** the test.
+
+4. **Publish** the test.
+
+5. From the Test list, **Run** the test. The notifications are disabled, so you won't receive an email notification.
+
+6. Edit the test and change the value for **Fact** from `true` to `false`.
+
+<img src={useBaseUrl('img/api-testing/fact-examples/disable-alerts-false.png')} alt="disable alerts false" />
+
+```yaml
+  - id: fact
+    identifier: disable_alerts
+    label: alerts disabled
+    value: "false"
+```
+
+7. **Save** the test.
+
+8. **Publish** the test.
+
+9. From the Test list, **Run** the test. The notifications are now activated, so you will receive an email notification.
+
+:::note
+In the example, we run the test manually but it works in the same way when you schedule it.
+:::
+
+## Setting Email Notification Thresholds
+
+You can use the **Fact** component to set the email alert threshold: if you want a test to fail more than once before an email is sent, a Fact called `mail_threshold` can be set in the test:
+
+<img src={useBaseUrl('img/api-testing/factMultiFailure.png')} alt="factMultiFailure.png" />
+
+This means the test will need to fail twice in a row before an email alert is sent.
+
+Given that this can be configured within the test, it offers all the flexibility provided by conditional statements, such as an **If** condition on the environment the test is running upon:
```

Comment:
```suggestion
Given that this can be configured in the test, it offers all the flexibility provided by conditional statements, such as an **If** condition on the environment the test is running upon:
```